thirdparty/README.md

```diff
@@ -16,6 +16,19 @@ Files extracted from upstream source:
 - `ffx_a.h` and `ffx_fsr1.h` from `ffx-fsr`
 - `license.txt`
 
+## amd-fsr2
```

Comment:
Probably license info should be added to https://github.com/godotengine/godot/blob/master/COPYRIGHT.txt as well, after fsr-1:

https://github.com/godotengine/godot/blob/df0a822323a79e1a645f0c6a17d51c7602f23166/COPYRIGHT.txt#L139-L142

Reply:
Gonna delegate to @akien-mga to give me the info on this one just in case.